test/sqllogictest/arithmetic.slt

```diff
@@ -317,3 +317,126 @@ query TTT
 SELECT 1.1::text, 1.1::float::text, 1.1::double::text
 ----
 1.1  1.1  1.1
+
+query R
+SELECT round(CAST(1.5678 as float))
+----
+2
+
+query R
+SELECT round(CAST(-1.4678 as float))
+----
+-1
+
+query R
+SELECT round(CAST(1.5678 as double precision))
+----
+2
+
+query R
+SELECT round(CAST(-1.4678 as double precision))
+----
+-1
+
+statement ok
+CREATE TABLE peeps (
+  peep integer
+)
+
+statement ok
+INSERT INTO peeps VALUES (4)
```

Comment:
Hi @lmatz, 
Sorry about the radio silence. We were away for a few days for an offsite, but now we're back!

Can you please add some tests to ensure proper behavior on null input?

`SELECT round(NULL)`
`SELECT round(NULL, <some scales>` 
`SELECT round(<some numbers>, NULL)`

And also, add value `NULL` to `peeps` and then add some tests where the peep column is the first argument of `round` and where the peep column is the second argument of `round`. This is because a NULL on its own is treated as NULL-type column, where as a NULL in an integer column is treated as an integer with value NULL.

Reply:
Sure, happy to add more tests!
Also check round(NULL, NULL) and when the peep column is `float`.